index.html

```diff
@@ -10,7 +10,47 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+    <header class="header__header"> 
```

Comment:
I'm impressed with how y'all reverse-engineered the example app! That's a skill in itself -- good work, team!